pkg/controller/sync/sync_controller.go

```diff
@@ -131,7 +131,7 @@ func (r *ReconcileSync) Reconcile(request reconcile.Request) (reconcile.Result,
 		return reconcile.Result{}, nil
 	}
 
-	r.log.Info("data will be added", "data", instance)
+	r.log.Debug("data will be added", "data", instance)
```

Comment:
It looks like `.Debug()` doesn't exist.

We could add a global constant `DebugLevel` to `pkg/logging/logging.go` with the value of 2, then evecute:

`r.log.V(logging.DebugLevel).Info(<ORIGNAL ARGS>)`

Reply:
this seems like a really strange logger if basic things like `Debug` don't exist :(

Reply:
updated

Reply:
Yeah, unclear to me why it's structured that way, it seems like something that'd be intentional though.